src/Compiler/Driver/fsc.fsi

```diff
@@ -13,18 +13,18 @@ open FSharp.Compiler.CodeAnalysis
 open FSharp.Compiler.Syntax
 open FSharp.Compiler.TcGlobals
 
-[<AbstractClass>]
+/// DiagnosticLoggers can be sensitive to the TcConfig flags. During the checking
+/// of the flags themselves we have to create temporary loggers, until the full configuration is
+/// available.
 type DiagnosticsLoggerProvider =
```

Comment:
nit: Since this is now an interface, we should prefix it with `I` -> `IDiagnosticsLoggerProvider` .